.github/workflows/golangci-lint.yml

```diff
@@ -32,20 +32,32 @@ jobs:
       contents: read  # for actions/checkout to fetch code
       pull-requests: read  # for actions/checkout to fetch pull requests
     name: lint
-    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # See #9943, we just need to add windows-latest here once all issues are fixed.
+        os: [ubuntu-latest, macos-latest]
+        go-version: ['1.19.4']
+    runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v2
+      - name: Checkout code
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v2
```

Comment:
Just curious as this is not the goal of the PR, but anything preventing us from using v3? (https://github.com/actions/checkout#checkout-v3) 

Same for actions/setup-go.

Reply:
Don't think so.

It's just that if we decide to upgrade them we should try upgrading them all at once (in all yaml actions). At least that's the way Guillaume did it in the past.